Makefile

```diff
@@ -24,8 +24,10 @@ LDFLAGS += -X "$(DASHBOARD_PKG)/pkg/utils.GitBranch=$(shell git rev-parse --abbr
 default:
 	SWAGGER=1 make server
 
-tidy:
-	go mod tidy
+lint:
```

Comment:
Need swagger_spec as a target?

Reply:
this may make it impossible to lint without swagger (swagger dependencies are always installed)?

Reply:
Sorry, I didn't see `docs/placeholder.go` before